common/main/java/com/couchbase/lite/Collection.java

```diff
@@ -136,16 +136,19 @@ static Collection getDefaultCollection(@NonNull Database db) throws CouchbaseLit
     // - Properties
 
     /**
-     * Return the collection name
+     * Get scope
      */
     @NonNull
-    public String getName() { return c4Collection.getName(); }
+    public Scope getScope() { return new Scope(c4Collection.getScope(), db); }
 
     /**
-     * Get scope
+     * Return the collection name
      */
     @NonNull
-    public Scope getScope() { return new Scope(c4Collection.getScope(), db); }
+    public String getName() { return c4Collection.getName(); }
+
+    @NonNull
+    public String getFullName() { return c4Collection.getScope() + "." + c4Collection.getName(); }
```

Comment:
Can it be non-public?

Reply:
Yes... but why?  You are about to request that it be public... That seems just so mean to our clients....

Reply:
I knew I want to have it public for all but not for one. Until then, I suggested that it is non-public.

Reply:
Done.  You are just a meany!!

Reply:
Sorry for that. I need to keep the rule that we all jump at the same time if possible.